arches/app/views/api.py

```diff
@@ -380,26 +389,34 @@ def post(self, request, resourceid=None):
             if user_can_edit_resources(user=request.user):
                 data = JSONDeserializer().deserialize(request.body)
                 reader = JsonLdReader()
-                reader.read_resource(data)
+                if slug is not None:
+                    graphid = models.GraphModel.objects.get(slug=slug).pk
+                reader.read_resource(data, graphid=graphid)
                 if reader.errors:
                     response = []
                     for value in reader.errors.itervalues():
                         response.append(value.message)
-                    return JSONResponse(data, indent=indent, status=400, reason=response)
+                    return JSONResponse({"error": response}, indent=indent, status=400)
                 else:
                     response = []
                     for resource in reader.resources:
                         with transaction.atomic():
                             resource.save(request=request)
+                        print resource.resourceinstanceid
```

Comment:
we can remove the print statement

Reply:
Probably? It's from the master branch which I had to pull in to deal with the django migration issue that had got out of line. Shall I do this here?

Reply:
Or maybe it was something Rob S put in... either way I can remove it.

Reply:
Removed

Reply:
Yep, there's lots of debugging still in there. Happy to remove most of it.